TodoListManagerTest/src/test/java/PerformanceTest/ProjectsPerformanceTest.java

```diff
@@ -0,0 +1,203 @@
+package PerformanceTest;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import org.json.JSONObject;
+import org.junit.jupiter.api.*;
+import java.io.IOException;
+import java.lang.management.ManagementFactory;
+import com.sun.management.OperatingSystemMXBean;
+@TestMethodOrder(MethodOrderer.Random.class)
+public class ProjectsPerformanceTest {
+    private static final String pUrl = "http://localhost:4567/projects";
+    private static ProcessBuilder pb;
+
+    private final String json = "application/json";
+    private final String xml = "application/xml";
+
+    private final int[] target = {1,5,10,50,100,200,500,1000};
+    private final int tot = target[target.length-1];
+
+    OperatingSystemMXBean osBean = (OperatingSystemMXBean) ManagementFactory.getOperatingSystemMXBean();
+
+    @BeforeAll
+    static void setupProcess() {
+        String os = System.getProperty("os.name");
+        if (os.toLowerCase().contains("windows")) {
+            pb = new ProcessBuilder(
+                    "cmd.exe", "/c", "java -jar .\\src\\test\\resources\\runTodoManagerRestAPI-1.5.5.jar");
+        }
+        else {
+            pb = new ProcessBuilder(
+                    "sh", "-c", "java -jar ./src/test/resources/runTodoManagerRestAPI-1.5.5.jar");
+        }
+    }
+
+    @BeforeEach
+    void startServer() throws InterruptedException {
+        try {
+            pb.start();
+            Thread.sleep(1000);
+        } catch (IOException e) {
+            System.out.println("No server");
+        }
+    }
+
+    @AfterEach
+    void shutServer() {
+        try {
+            RestAssured.get("http://localhost:4567/shutdown");
+        }
+        catch (Exception ignored) {
+        }
+    }
+
+
+    @Test
+    public void createProjectsJsonPut() {
```

Comment:
pretty sure that for creating you need Post

Reply:
Sure. I will change it.